src/main/java/frc/robot/vision/data/AprilTagVisionData.java

```diff
@@ -4,23 +4,31 @@
 
 public class AprilTagVisionData extends VisionData {
 
+	private final double[] standardDeviations;
 	private final double aprilTagHeightMeters;
 	private final double distanceFromAprilTagMeters;
 	private final int aprilTagId;
 
 	public AprilTagVisionData(
+		String sourceName,
 		Pose3d estimatedRobotPosition,
 		double timestamp,
+		double[] standardDeviations,
```

Comment:
it requires 6 dims. Has to be clear from the name. The only reason I know it requires xyz and 3D ang is because I asked for that field ;)